### ISSUE TITLE
Unsupported readlink -e option in busybox

### ISSUE THREAD

**Issue by vk496** [digint/btrbk#327]
Hi. My ssh backend have a readlink through busybox, which don't have a `-e` option. What should/could I do?

```
btrbk command line client, version 0.29.1  (Sun Apr 19 21:32:40 2020)
Using configuration: nas.conf
WARNING: Skipping target "nas.lan:/remote/path": Failed to fetch subvolume detail
WARNING: ... Command execution failed (exitcode=1)
WARNING: ... sh: ssh -i /home/vk496/.ssh/nas vk496@nas.lan 'readlink -v -e /remote/path'
WARNING: ... readlink: invalid option -- 'e'
WARNING: ... BusyBox v1.19.3 (2020-03-24 01:15:42 CST) multi-call binary.
WARNING: ... 
WARNING: ... Usage: readlink [-fnv] FILE
WARNING: ... 
WARNING: ... Display the value of a symlink
WARNING: ... 
WARNING: ...    -f      Canonicalize by following all symlinks
WARNING: ...    -n      Don't add newline
WARNING: ...    -v      Verbose
WARNING: ... 
Assuming non-present subvolume "vk496.20200419" in skipped targets: "nas.lan:/remote/path"
Creating subvolume snapshot for: /home/vk496
[snapshot] source: /home/vk496
[snapshot] target: /home/vk496/btrbk/btrbk_snapshots/vk496.20200419
WARNING: Skipping cleanup of snapshots for subvolume "/home/vk496", as at least one target aborted earlier
Completed within: 0s  (Sun Apr 19 21:32:40 2020)
--------------------------------------------------------------------------------
Backup Summary (btrbk command line client, version 0.29.1)

    Date:   Sun Apr 19 21:32:40 2020
    Config: nas.conf
    Dryrun: YES

Legend:
    ===  up-to-date subvolume (source snapshot)
    +++  created subvolume (source snapshot)
    ---  deleted subvolume
    ***  received subvolume (non-incremental)
    >>>  received subvolume (incremental)
--------------------------------------------------------------------------------
/home/vk496
+++ /home/vk496/btrbk/btrbk_snapshots/vk496.20200419
!!! Target "nas.lan:/remote/path" aborted: Failed to fetch subvolume detail

NOTE: Some errors occurred, which may result in missing backups!
Please check warning and error messages above.

NOTE: Dryrun was active, none of the operations above were actually executed!
```

My config:

```
snapshot_preserve_min       2d
snapshot_preserve          14d

target_preserve_min        no
target_preserve            20d 10w *m

snapshot_dir               vk496/btrbk/btrbk_snapshots

ssh_identity               /home/vk496/.ssh/nas
ssh_user                   vk496

volume /home
    subvolume vk496
    target send-receive ssh://nas.lan/remote/path
```

PS: `/home` don't have any subvolumes, but if I don't specify a `suvolume`, it not doing anything.

Thnks in advance

**Comment by digint** [comment]
Thanks for reporting (and sorry for the late reply...)

I fixed this in the [compat-busybox branch](https://github.com/digint/btrbk/tree/compat-busybox)  (commit 8a4257fba45bbc23609c14c2866e5e138c8f11ad). Needs some more testing before being merged into master.

**Comment by digint** [comment]
Merged to master: ec037952cf355c4107c03a13d7ad29678e39fa36

**Comment by digint** [comment]
Included in [btrbk-0.30.0](https://github.com/digint/btrbk/releases/tag/v0.30.0)
